### PR TITLE
Style Fixes - Mobile View

### PR DESCRIPTION
**Pull request by IrynaVynnychenko**
Hello. :)
I've finished fix styles of the app, may you look, please, if everything okay...
If no, let me know, I may fix anything, that need. )

### CODE REVIEW

File: dapp/src/components/page/home/MoreModal/MoreModal.scss [hunk 1]
```diff
@@ -7,6 +7,10 @@
   border: 1px solid rgb(235, 235, 235);
   box-shadow: 0px 0px 60px rgba(82, 82, 82, 0.1);
   border-radius: 12px;
-  padding: 12px 12px;
+  padding: 8px 12px;
```
Comment (w4ll3) [inline dapp/src/components/page/home/MoreModal/MoreModal.scss:10]: Again, if possible move to Tailwind classes

File: dapp/src/components/page/view/PublicProfileView.svelte [hunk 2]
```diff
@@ -51,14 +51,14 @@
     </div>
   {/if}
   <div class="flex flex-row items-center">
-    <div class="flex flex-row items-center cursor-pointer bubble-outline">
-      <div class="address-container">
-        {$params.address}
+    <div class="flex flex-row items-center cursor-pointer bubble-outline mr-0">
+      <div class="address-container mr-2">
+        {$params.address.slice(0,5)}...{$params.address.slice(-4)}
```
Comment (w4ll3) [inline dapp/src/components/page/view/PublicProfileView.svelte:56]: Second time I'm seeing this function, please consider a component

File: dapp/src/routes/Search/search.scss [hunk 3]
```diff
@@ -1,6 +1,9 @@
 .search-container{
-    max-width: 800px;
+    max-width: 732px;
+    width: 100%;
```
Comment (w4ll3) [inline dapp/src/routes/Search/search.scss:3]: Consider removing this file and using Tailwind classes

File: dapp/src/routes/Splash/splash.scss [hunk 4]
```diff
@@ -1,11 +1,16 @@
-.splash-container{
+.splash-container {
     margin: 50px auto;
     max-width: 800px;
     display: flex;
     flex-direction: column;
     padding: 24px;
+
```
Comment (w4ll3) [inline dapp/src/routes/Splash/splash.scss:7]: Remove file and use Tailwind classes

Comment (IrynaVynnychenko) [reply dapp/src/routes/Splash/splash.scss]: I've just remade scss to tailwind css classes, files have not deleted. Is it okay?
I may also clean html, add name of the classes and remove all tailwind classes in files, like these, to each component, if need.
May you look, please if it's okay.

File: dapp/src/components/page/view/PublicProfileView.svelte (hunk 2 above)
Comment (IrynaVynnychenko) [reply dapp/src/components/page/view/PublicProfileView.svelte:56]: {$userData.account.address.slice(0,5)}...{$userData.account.address.slice(-4)}
----
{$params.address.slice(0,5)}...{$params.address.slice(-4)}

different info for each time, it's not same, maybe better not to create separate component for each one?:)

File: dapp/src/routes/Splash/splash.scss (hunk 4 above)
Comment (w4ll3) [reply dapp/src/routes/Splash/splash.scss]: Great!

File: dapp/src/components/page/view/PublicProfileView.svelte (hunk 2 above)
Comment (w4ll3) [reply dapp/src/components/page/view/PublicProfileView.svelte:56]: Mmmm, I thought I might've seen that somewhere else, my bad.

File: dapp/src/components/modals/FileModal.svelte [hunk 5]
```diff
@@ -12,12 +12,19 @@
     background-color: white;
     max-height: 80vh;
     overflow: auto;
+    margin-left: 16px;
+    margin-right: 16px;
+    padding: 40px 20px;
```
[the remark this answers is not in the document]
Comment (IrynaVynnychenko) [reply dapp/src/components/modals/FileModal.svelte:17]: add fixes, removed css from html files

**Pull request merged**